policies/faq.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Frequently Asked Questions (FAQ)
+---
+
+#### How big can teams be?
+
+Up to 5 people, if you're looking to submit your project for the weekly and/or final prizes. Otherwise, we don't really mind if you go over, so as long as your project sits out from prizing considerations!
+
+#### Is there a minimum age requirement?
+
+Yes, you must be **13 years of age or older** at the time of entry. Being a global hackathon, we have to comply with many policies around the world, such as the FTC's [COPPA](https://www.ftc.gov/tips-advice/business-center/guidance/childrens-online-privacy-protection-rule-six-step-compliance).
+
+#### What about Intellectual Property (IP)?
+
+*   The intellectual property of each project is owned by their respective teams.
+*   Teams that use someone else's assets (e.g. libraries, frameworks, source code, images, etc.) are responsible for abiding by each asset's usage rights. Failure to do so will **automatically disqualify** that team from current and future prize consideration.
+*   In case it wasn't clear, no stealing of others' intellectual property! Be 100% sure you are allowed to use something before you do.
+*   While not required during submission, teams must be ready to provide proof of fair use of intellectual property to judges and organizers upon request.
+
+#### Does our solution need to be open-source?
+
+*   No, not necessarily.
+*   For teams that choose to keep their solution proprietary, judges and organizers reserve the right to scrutinize the source code & other assets for compliance with usage rights.
+*   For all teams, failure to allow judges' and organizers' scrutiny may result in disqualification from prizing of any nature.
+
+#### Can I work on a project I've already started before CODEVID-19?
+
+That's fine, however only projects that were started **on or after March 16, 00:00 UTC** can be submitted for prizing considerations. See [Submissions](#submission) for more details.
+
+#### When do sign-ups start? When do they end?
+
+Sign-ups have started! Register on [FindCollabs](https://findcollabs.com/hackathon/codevid-19-isp21fkqtjupchx7kjed) and [join](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) our [Slack](https://codevid-19.slack.com). We will close sign-ups one week prior to the [final submission date](#submission).
+
+#### Do you have datasets and other resources we could start from?
+
+Volunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!
```

Comment:
"olunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!" -> "We're working on a pre-compiled list, but until it's available you can find many tools posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join" (Personal preference, it's still good as it is)

Reply:
Made ze update, Mandy let me know that if you're already registered and you click the invite link, Slack will automatically redirect you to our workspace, so I condensed your recommendation even further by chopping the last bit